rpcs3/Emu/Cell/Modules/cellMsgDialog.cpp

```diff
@@ -489,9 +504,10 @@ error_code cellMsgDialogAbort()
 
 error_code cellMsgDialogOpenSimulViewWarning(vm::ptr<CellMsgDialogCallback> callback, vm::ptr<void> userData, vm::ptr<void> extParam)
 {
-	cellSysutil.todo("cellMsgDialogOpenSimulViewWarning(callback=*0x%x, userData=*0x%x, extParam=*0x%x)", callback, userData, extParam);
+	cellSysutil.warning("cellMsgDialogOpenSimulViewWarning(callback=*0x%x, userData=*0x%x, extParam=*0x%x)", callback, userData, extParam);
```

Comment:
this will not change after you changed the string. keep it on todo

Reply:
What would be left 'todo' about this function?

From the PS3 sample code (as executed on a PS3), it just displays the message 'dialog' warning, doesn't allow it to be skipped, and automatically closes it again after a number of seconds.
As noted in my PR comment, I don't believe we would want the "doesn't allow it to be skipped" part, so it does seem to fulfill what I would consider as 'our requirements' for the cellMsgDialoOpenSimulViewWarning function.

Reply:
why would we not want it?
It's how it should behave. you don't know if some developers (or maybe even the utility) relied on a time frame or not.

Reply:
Ok, skip capability removed.